extensions/kubernetes-client/runtime/src/main/java/io/quarkus/kubernetes/client/runtime/KubernetesClientProducer.java

```diff
@@ -34,7 +34,7 @@ public Config config() {
                 .withCaCertFile(buildConfig.caCertFile.orElse(base.getCaCertFile()))
                 .withCaCertData(buildConfig.caCertData.orElse(base.getCaCertData()))
                 .withClientCertFile(buildConfig.clientCertFile.orElse(base.getCaCertFile()))
```

Comment:
I assume this line needs to be fixed as well?

Reply:
Yes, you are right. Fix it as well.